blocks.md

```diff
@@ -288,12 +290,27 @@ Key | Values | Required | Default
 ----|--------|----------|--------
 `path` | Path to collect information from | No | `"/"`
 `alias` | Alias that is displayed for path | No | `"/"`
-`info_type` | Currently supported options are `"available"`, `"free"`, `"total"` and `"used"` | No | `"available"`
+`icon` | Whether to prepend the block with the hard disk icon | No | `false`

```

Comment:
We can remove this config option by allowing `icon` to be set in the format string.
See the KDEConnect block for something similar.

Reply:
I originally did it that way, but does it make sense since we have the `.with_icon()` method and the icon would always be at the front anyway?

It's a simple change in any case, I just didn't see much of a use case in putting the icon anywhere but the front (in which case we can use the existing method, and do it outside the `update()` call.

Reply:
I think I'd rather it go in the format string especially since it was originally proposed as an alternative to the alias.